draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3456,808 +3548,1280 @@ ObjectLists MAY combine regular objects and secondary ObjectLists in the same ob
-   Status Resources shown in the example above, to be updated to:</t>
 
-   <figure><artwork><![CDATA[
+    </section>
+    <section anchor="trigger-with-extensions" title="Extensions with Error Propagation">
+    <t>
+    In the following example, a CI/T "preposition" command is using two extensions to control the
+    "AS64500:0", does not support the first extension in the extensions array. dCDN "AS64500:0" further
+    distributes this trigger to another downstream CDN that is identified as "AS64501:0", which
+    does not support the second extension in the extensions array. The error is propagated from
+    "AS64501:0" to "AS64500:0" and the errors.v2 array reflects both errors.
+    </t>
+    <figure>
+    <artwork><![CDATA[
 REQUEST:
+  
+POST /cit HTTP/1.1
+User-Agent: example-user-agent/0.1
+Host: dcdn.example
+Accept: */*
+Content-Type: application/cdni; ptype=ci-trigger.v2
+Content-Length: 1249
 
-  GET /triggers/complete HTTP/1.1
-  User-Agent: example-user-agent/0.1
-  Host: dcdn.example.com
-  Accept: */*
+{
+  "action": "preposition",
+  "specs": [{
+    "trigger-subject": "content",
+    "generic-trigger-spec-type": "content-objectlist",
+    "generic-trigger-spec-value": {
+      "objects": [{
+        "href": "https://www.example.com/hls/title/index.m3u8",
+        "type": "hls"
+      }]
+    }
+  }],
+  "extensions": [
+    {
+      "generic-trigger-extension-type": "location-policy",
+      "generic-trigger-extension-value": {
+        "locations": [
+          {
+            "action": "allow",
+            "footprints": [{
+                "footprint-type": "countrycode",
+                "footprint-value": [ "us" ]
+            }]
+          },
+          {
+            "action": "deny",
+            "footprints": [{
+               "footprint-type": "countrycode",
+               "footprint-value": [ "ca" ]
+            }]
+          }
+        ]
+      },
+      "mandatory-to-enforce": true,
+      "safe-to-redistribute": true
+    },
+    {
+      "generic-trigger-extension-type": "time-policy",
+      "generic-trigger-extension-value": {
+        "unix-time-window": { 
+          "start": 1730174400, 
+          "end": 1730260800 
+        }   
+      },
+      "mandatory-to-enforce": true,
+      "safe-to-redistribute": true
+    }
+  ],
+  "cdn-path": [ "AS64496:0" ]
+}
 
 RESPONSE:
 
-  HTTP/1.1 200 OK
-  Content-Length: 105
-  Expires: Wed, 04 May 2016 08:49:22 GMT
-  Server: example-server/0.1
-  ETag: "-6938620031669085677"
-  Cache-Control: max-age=60
-  Date: Wed, 04 May 2016 08:48:22 GMT
-  Content-Type: application/cdni; ptype=ci-trigger-collection
-
-  {
-    "staleresourcetime": 86400,
-    "triggers": [ "https://dcdn.example.com/triggers/1" ]
-  }
-]]></artwork>
-   </figure>
-   </section>
+HTTP/1.1 201 Created
+Date: Sun, 27 Oct 2024 08:48:31 GMT
+Content-Length: 2595
+Content-Type: application/cdni; ptype=ci-trigger.v2
+Location: https://dcdn.example/cit/3f2d259d-a980-4742-beeb-9392a58129f5
```

Comment:
Nit: We may wish to choose a different UUID, as this technically references a different example.